legal-api/src/legal_api/models/business.py

```diff
@@ -329,6 +329,8 @@ def business_legal_name(self):
     @property
     def next_anniversary(self):
         """Retrieve the next anniversary date for which an AR filing is due."""
+        if not self.founding_date and not self.last_ar_date:
```

Comment:
I found a bunch of corps in COLIN not have founding date, so I've made some tweaks to make sure basic API calls work for these corps. The changes may be not thorough.